AirSkyBoat/AirSkyBoat
# Implements Starlight Festival 2021

**_I affirm:_**

## What does this pull request do?

Implements most of the Starlight Celebration Events.

- Starlight Decorations in Bastok/San d'Oria/Windurst
- Starlight Theme Music in Jeuno (via sql module)
- Gifts for NPCs Mini quest
- Goblin Merrymakers Mini Quest*
- Smilebringers Fame Mini Quest/Dream Hat +1
- Smilebringer Bootcamp Quests
- Holiday Token Redemption Questline
- Initial work on additional decorations



## Steps to test these changes

This relies on updating your settings/main.lua file as this is ignored. Settings should be set like so:

Under CELEBRATIONS:
`
STARLIGHT_2021 = 1
STARLIGHT_YEAR_AROUND = optional
`


Enable via Settings and have a Merry Starlight!
